man/make.conf.5

```diff
@@ -948,6 +948,14 @@ Setting this and other *FLAGS variables arbitrarily may cause compile or
 runtime failures. Bug reports submitted when nonstandard values are
 enabled for these flags may be closed as INVALID.
 .TP
+.B MAKEFLAGS
+Use this variable instead of \fBMAKEOPTS\fR if you want to inject a
+centralized job server for make. In this case \fBMAKEOPTS\fR should be
+unset or else it can cause the make jobserver mode to reset. Include
+a \fB--jobserver-auth=fifo:PATH\fR flag to specify the path of the
+centralized jobserver fifo, which needs to be readable and writable by
+the portage group when userpriv is enabled.
+.TP
```

Comment:
If we parse MAKEFLAGS then we can automatically add a sandbox exemption for the jobserver fifo. Otherwise users will have to do this themselves or put the fifo in an unsandboxed location like $PORTAGE_TMPDIR (they might have to do that for `sesandbox` too).

Reply:
> > > Wouldn't this pass option like `-k` though, too?
> > 
> > 
> > Yes, in fact -k is considered a "simple" option because it has no argument, and make represents these simple options as a group of letters at the beginning of MAKEFLAGS like `MAKEFLAGS="k -j4 -l4 --jobserver-auth=fifo:/tmp/GMfifo30205"`.
> > If there are no simple options then MAKEFLAGS begins with a space.
> 
> I mean, passing arbitrary options sounds dangerous and undesirable. If someone currently runs `emerge` via `make -k` as part of some bigger script, I don't think we should pass that one to make invocations within package builds.

If we parse MAKEFLAGS in order to add a sandbox exemption for the fifo, I suppose we might filter out the simple options while we're there.